mathics_scanner/tokeniser.py

```diff
@@ -378,7 +378,7 @@ def __eq__(self, other):
         )
 
     def __repr__(self):
-        return f"Token({self.tag}, {self.text}, {self.pos})"
+        return f"Token({repr(self.tag)}, {repr(self.text)}, {self.pos})"
```

Comment:
This is a slightly nicer repr which makes clear which parts (tag and text) are strings.